modern/src/MainPage.js

```diff
@@ -36,39 +41,205 @@ const useStyles = makeStyles((theme) => ({
+          className={classes.deviceButton}
+          onClick={handleClose}
+          startIcon={<ListIcon />}
+          disableElevation
         >
-          <DevicesList />
-        </Drawer>
-        <div className={classes.mapContainer}>
-          <ContainerDimensions>
-            <Map>
-              <CurrentLocationMap />
-              <GeofenceMap />
-              <AccuracyMap />
-              <CurrentPositionsMap />
-              <SelectedDeviceMap />
-            </Map>
-          </ContainerDimensions>
-        </div>
+          {!matchesMD ? t('deviceTitle') : ''}
+        </Button>
+        )}
+      <div className={`${classes.listContainer} ${collapsed ? classes.collapsed : classes.uncollapsed}`}>
+        <Grid container direction="column" spacing={matchesMD ? 0 : 2} style={{ height: '100%' }}>
+          <Grid item>
+            <Paper className={classes.paper}>
+              <Toolbar className={classes.toolbar} disableGutters>
+                <Grid container direction="row" alignItems="center" spacing={2}>
+                  {matchesMD && (
+                  <Grid item>
+                    <IconButton onClick={handleClose}>
+                      <ArrowBackIcon />
+                    </IconButton>
+                  </Grid>
+                  )}
+                  <Grid item xs>
+                    <TextField
+                      fullWidth
+                      name="deviceName"
+                      value={deviceName || ''}
+                      autoComplete="deviceName"
+                      autoFocus
+                      onChange={(event) => setDeviceName(event.target.value)}
+                      placeholder="Search Devices"
+                      variant="filled"
+                    />
+                  </Grid>
+                  <Grid item>
+                    <IconButton onClick={() => history.push('/device')}>
+                      <AddIcon />
+                    </IconButton>
+                  </Grid>
+                  {!matchesMD && (
+                  <Grid item>
+                    <IconButton onClick={handleClose}>
+                      <CloseIcon />
+                    </IconButton>
+                  </Grid>
+                  )}
+                </Grid>
+              </Toolbar>
+            </Paper>
+          </Grid>
+          <Grid item xs>
+            <Paper className={`${classes.deviceList} ${classes.paper}`}>
+              <DevicesList />
+            </Paper>
+          </Grid>
+        </Grid>
+      </div>
+      <div className={classes.bottomMenuContainer}>
+        <Paper className={classes.paper}>
+          <Toolbar className={classes.toolbar} disableGutters>
+            <Grid container justify="space-around">
+              <Grid item>
+                <IconButton classes={{ label: classes.menuButton }}>
+                  <ReplayIcon />
+                  <span className={classes.iconText}>{t('reportReplay')}</span>
```

Comment:
Can we apply the text style to the whole container?

Reply:
Done.